test/unit-tests/driver/driver-codemodel-tests.ts

```diff
@@ -171,25 +175,27 @@ export function makeCodeModelDriverTestsuite(driverName: string, driver_generato
                 .to.eq(path.normalize(path.join(root, 'test_project', 'static_lib_dummy')).toLowerCase());
 
             // Language
-            const compile_information = target!.fileGroups!.find(t => !!t.language);
+            const compile_information = target!.fileGroups!.find((t: any) => !!t.language);
             expect(compile_information).to.be.not.undefined;
             expect(compile_information!.language).to.eq('CXX');
 
             // Test main source file
             expect(compile_information!.sources).to.include('info.cpp');
             expect(compile_information!.sources).to.include('test2.cpp');
 
-            // compile flags for file groups
+            // compile flags or fragments for file groups
             if (process.platform === 'win32') {
-                expect(compile_information!.compileFlags?.trim()).to.eq('/DWIN32 /D_WINDOWS /W3 /GR /EHsc /MDd /Zi /Ob0 /Od /RTC1');
+                if (compile_information) {
```

Comment:
You should assert `compile_information!` like the test used to instead of changing this to `if`.